flexibee-core/src/main/java/com/adleritech/flexibee/core/api/FlexibeeClient.java

```diff
@@ -89,30 +83,40 @@ private void handleErrorResponse(Response response) throws FlexibeeException {
 
     private void handleErrorResponse(Response response, WinstromRequest winstromRequest) throws FlexibeeException {
         if (!response.isSuccessful()) {
-            String request = "n/a";
-            if( winstromRequest != null) {
-                StringWriter writer = new StringWriter(200);
+            if (response.code() == 404) {
```

Comment:
Proc je 404 tak zvlastni, ze se u ni nezachovava rawErrorResponse?

Reply:
Vlastně není. Tahle odbočka tam byla a já jí nechal tak jak byla. Pravděpodobně tam byla proto aby se v tomhle případě celé to xml nelogovalo. Není důvod tam tu rawResponse nedat.

Reply:
:+1: